cellengine/resources/fcs_file.py

```diff
@@ -2,7 +2,11 @@
 from cellengine.utils import readonly
 
 from attr import define, field
-from typing import Any, Dict, List, Literal, Optional, Union
```

Comment:
Looks like this commit (`src(apiclient): import Literal for Python <3.8`) should be squashed with the one where Literal is first added. (Commits should never be merged to the main branch with known bugs.)

Reply:
:+1: